### ISSUE TITLE
LINUX --cfg : trailing space leads to WARNING: failed to open cfg file

### ISSUE THREAD

**Issue by PYB-SU** [vdemichev/DiaNN#641]
Hi Vadim,
Coming back to a problem that was already mentionned but still present in 1.8.1.

In arguments() on linux, name contains a trailing space which leads to failure to open file.
You may fix this for all LINUX users...

#ifdef LINUX
                  std::string name = args.substr(start + 4, end - start - 5);
#else
                  std::string name = args.substr(start + 4, end - start - 4);
#endif

(removing the space may perhaps work on windows as well, I didn't test it)

Thanks a lot.

**Comment by PYB-SU** [comment]
even better : 

std::string name = trim(args.substr(start + 4, end - start - 4));

**Comment by PYB-SU** [comment]
it occured to me overnight that a VERY DIRTY workaround for LINUX is to change the name of the config file to include a trailing space... 

cp cfg.txt cfg.txt\      # note last character is \" " 

then 

diann --cfg cfg.txt  # here do not include the trailing space

not that I recommend it.

**Comment by vdemichev** [comment]
Yes, this is a known bug on Linux, will be fixed in the next version. There's a simple workaround: add '--' directly after the file name, i.e. "--cfg ./config.txt--" (without quotes)
